gtk/DetailsDialog.cc

```diff
@@ -805,13 +805,13 @@ void DetailsDialog::Impl::refreshInfo(std::vector<tr_torrent*> const& torrents)
         {
             str = mixed;
         }
-        else if (baseline < 0)
+        else if (baseline == TR_ETA_NOT_AVAIL || baseline == TR_ETA_UNKNOWN)
```

Comment:
I prefer `baseline < 0`: we don't want to show any negative value anyway.

Reply:
There are no negative values anymore; you changed the type to a time_t :smile_cat:

Reply:
I'm reading https://en.wikipedia.org/wiki/Year_2038_problem now.
While most systems went from a **signed** 32 bit integer to a **signed** 64 bit integer to fix the Year 2038 problem, some rare systems may have quirkly adopted an unsigned 32 bit integer to buy some time.

Reply:
You're right!

Reply:
Updated 5ee59a4dc3ed578e2de1f355769d96fd20dc3d40